### PR TITLE
update modules list for akka enhancements

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: akka-docs/src/main/paradox/guide/modules.md [hunk 1]
```diff
@@ -15,15 +15,21 @@ The following capabilities are included with Akka OSS and are introduced later o
 * [Streams](#streams)
 * [HTTP](#http)
 
-With a Lightbend subscription, you can use [Enterprise Suite](https://www.lightbend.com/platform/production) in production. Enterprise Suite includes the following extensions to Akka core functionality:
+With a Lightbend subscription, you can use [Akka Enhancements](https://doc.akka.io/docs/akka-enhancements/current/) that includes:
```
Comment (raboof) [inline akka-docs/src/main/paradox/guide/modules.md]:
```suggestion
With a [Lightbend Platform Subscription](https://www.lightbend.com/platform/subscription), you can use [Akka Enhancements](https://doc.akka.io/docs/akka-enhancements/current/) that includes:
```

File: akka-docs/src/main/paradox/guide/modules.md [hunk 2]
```diff
@@ -15,15 +15,21 @@ The following capabilities are included with Akka OSS and are introduced later o
...
+
+Resilience Enhancements:
```
Comment (raboof) [inline akka-docs/src/main/paradox/guide/modules.md]:
```suggestion
[Akka Resilience Enhancements](https://doc.akka.io/docs/akka-enhancements/current/akka-resilience-enhancements.html):
```

File: akka-docs/src/main/paradox/guide/modules.md [hunk 3]
```diff
@@ -15,15 +15,21 @@ The following capabilities are included with Akka OSS and are introduced later o
...
-* [Fast Failover](https://doc.akka.io/docs/akka-enhancements/current/fast-failover.html) &#8212; Fast failover for Cluster Sharding.
 * [Configuration Checker](https://doc.akka.io/docs/akka-enhancements/current/config-checker.html) &#8212; Checks for potential configuration issues and logs suggestions.
 * [Diagnostics Recorder](https://doc.akka.io/docs/akka-enhancements/current/diagnostics-recorder.html) &#8212; Captures configuration and system information in a format that makes it easy to troubleshoot issues during development and production.
 * [Thread Starvation Detector](https://doc.akka.io/docs/akka-enhancements/current/starvation-detector.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
+* [Kubernetes Lease](https://doc.akka.io/docs/akka-enhancements/current/kubernetes-lease.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
+* [Fast Failover](https://doc.akka.io/docs/akka-enhancements/current/fast-failover.html) &#8212; Fast failover for Cluster Sharding.
+
+Persistence enhancements:
```
Comment (raboof) [inline akka-docs/src/main/paradox/guide/modules.md]:
```suggestion
[Akka Persistence Enhancements](https://doc.akka.io/docs/akka-enhancements/current/akka-persistence-enhancements.html):
```

**Review by raboof — APPROVED** [review]

File: akka-docs/src/main/paradox/typed/guide/modules.md [hunk 4]
```diff
@@ -14,15 +14,21 @@ The following capabilities are included with Akka OSS and are introduced later o
 * [Streams](#streams)
 * [HTTP](#http)
 
-With a Lightbend subscription, you can use [Enterprise Suite](https://www.lightbend.com/platform/production) in production. Enterprise Suite includes the following extensions to Akka core functionality:
+With a Lightbend subscription, you can use [Akka Enhancements](https://doc.akka.io/docs/akka-enhancements/current/) that includes:
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/guide/modules.md]:
```suggestion
With a [Lightbend Platform Subscription](https://www.lightbend.com/platform/subscription), you can use [Akka Enhancements](https://doc.akka.io/docs/akka-enhancements/current/) that includes:
```

File: akka-docs/src/main/paradox/typed/guide/modules.md [hunk 5]
```diff
@@ -14,15 +14,21 @@ The following capabilities are included with Akka OSS and are introduced later o
...
+
+Resilience Enhancements:
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/guide/modules.md]:
```suggestion
[Akka Resilience Enhancements](https://doc.akka.io/docs/akka-enhancements/current/akka-resilience-enhancements.html):
```

File: akka-docs/src/main/paradox/typed/guide/modules.md [hunk 6]
```diff
@@ -14,15 +14,21 @@ The following capabilities are included with Akka OSS and are introduced later o
...
-* [Fast Failover](https://doc.akka.io/docs/akka-enhancements/current/fast-failover.html) &#8212; Fast failover for Cluster Sharding.
 * [Configuration Checker](https://doc.akka.io/docs/akka-enhancements/current/config-checker.html) &#8212; Checks for potential configuration issues and logs suggestions.
 * [Diagnostics Recorder](https://doc.akka.io/docs/akka-enhancements/current/diagnostics-recorder.html) &#8212; Captures configuration and system information in a format that makes it easy to troubleshoot issues during development and production.
 * [Thread Starvation Detector](https://doc.akka.io/docs/akka-enhancements/current/starvation-detector.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
+* [Kubernetes Lease](https://doc.akka.io/docs/akka-enhancements/current/kubernetes-lease.html) &#8212; Monitors an Akka system dispatcher and logs warnings if it becomes unresponsive.
+* [Fast Failover](https://doc.akka.io/docs/akka-enhancements/current/fast-failover.html) &#8212; Fast failover for Cluster Sharding.
+
+Persistence enhancements:
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/guide/modules.md]:
```suggestion
[Akka Persistence Enhancements](https://doc.akka.io/docs/akka-enhancements/current/akka-persistence-enhancements.html):
```

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]
LGTM with raboofs changes

**Pull request merged**